### DISCUSSION TITLE
Update Luau version

### DISCUSSION THREAD

**Discussion by filiptibell** [mlua-rs/mlua#247] [Ideas]
Hi, I've been using mlua for a bit in my personal project [Lune](https://github.com/filiptibell/lune) which is a standalone Luau script runtime, and some users are asking for a more recent Luau version to support the interpolated strings feature which was stabilized a few months ago (blog post about it here https://luau-lang.org/2023/02/02/luau-string-interpolation.html).

Could Luau be updated to the latest release for mlua `0.9` which is coming soon?
Also, would it be possible to let mlua crate consumers specify their own Luau version? Assuming the interface is stable

**Comment by khvzak** [comment]
You could run `cargo update` in your project to get the latest Luau version (0.561 for now) which supports string interpolation.

> Also, would it be possible to let mlua crate consumers specify their own Luau version? Assuming the interface is stable

You can add `luau0-src` crate to your project dependencies with desired version.
